tests/bluetooth/init/prj_ctlr_central priv.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_BT=y
```

Comment:
Use underscore instead of space in filename

Reply:
must be something wrong with my keyboard :P